domain-model-assistant/Assets/Components/Scripts/Position.cs

```diff
@@ -27,3 +27,22 @@ public class CreateJson
 
     public float y; 
 }
+
+
+public class AddAttributeInfo
```

Comment:
`...Info` is a software engineering term

Reply:
Renamed the class